packages/client/components/annotation-indicators/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import styled from 'styled-components'
+import { AnnotationIcon, AddAnnotationIcon } from '../icons'
+
+export interface Props {
```

Comment:
I think we should stay consistent with the naming style, i.e. AnnotationIndicatorsProps.

Reply:
I probably forgot about this a million times already.